lib/vm/src/vmcontext.rs

```diff
@@ -340,7 +340,7 @@ pub struct VMMemoryDefinition {
     pub base: *mut u8,
 
     /// The current logical size of this linear memory in bytes.
-    pub current_length: u32,
+    pub current_length: u64,
```

Comment:
I wonder if we should not create a type alias to represent the memory size, something like `pub type MemorySize = u64`. Thoughts @nlewycky, @MarkMcCaskey, @syrusakbary?

Reply:
I don't think so in this case, we aren't going to be changing this type or confusing it with other values much.